pages/Documents/ConversationalAI/Third-PartyBots/CustomEndpoint/basic-content.md

```diff
@@ -0,0 +1,506 @@
+        "confidenceScore": 1
+      }
+    ]
+  }
+}
+```
+
+Figure 2.5 Showing an example response of Structure Content Message
+
+### Sending Quick Replies (Structured Content)
+
+{: .important}
+**Please note** Quick Replies are only supported in Messaging Conversations.
+
+Quick Replies is a special type of Structured Content. It is a message sent along with predefined answers.
+For detailed information on Quick Replies check out the documentation for the specific channel ([Mobile SDK and Web](mobile-sdk-and-web-templates-quick-replies-template.html), [Facebook Messenger](facebook-messenger-templates-quick-replies-template.html),
+[Google RCS Business Messaging](google-rcs-business-messaging-templates-quick-replies-template.html)).
+
+An example of a Quick Replies message response along with any optional metadata and analytical
+information (intents) can be seen below in Figure 2.6. More information on responses expected
+by Third-Party Bot Connector can be found at [API Service Specification](https://github.com/LivePersonInc/third-party-bots-custom-endpoint-reference-service)
+
+{: .important}
+The response is expected to be sent by the [Get Conversation Events](third-party-bots-custom-endpoint-basic-content.html#custom-endpoint-ce-service-methods)
+method of the Custom Endpoint service please ensure valid structured content is sent by using our
+[Conversation Tester feature](third-party-bots-conversation-tester.html) or using
+[this tool](https://livepersoninc.github.io/json-pollock/editor/).
+
+```json
+{
+  "response": [
+    {
+      "type": "STRUCTURED_CONTENT",
+      "data": {
+        "structuredContent": {
+          "quickReplies": {
+            "type": "quickReplies",
+            "itemsPerRow": 8,
+            "replies": [
+              {
+                "type": "button",
+                "tooltip": "yes i do",
+                "title": "yes",
+                "click": {
+                  "actions": [{ "type": "publishText", "text": "yep" }],
+                  "metadata": [{ "type": "ExternalId", "id": "Yes-1234" }]
+                }
+              },
+              {
+                "type": "button",
+                "tooltip": "No!",
+                "title": "No!",
+                "click": {
+                  "actions": [{ "type": "publishText", "text": "No!" }],
+                  "metadata": [{ "type": "ExternalId", "id": "No-4321" }]
+                }
+              }
+            ]
+          },
+          "message": "Do you like Bots?"
+        },
+        "metadata": [{ "id": "1234", "type": "ExternalId" }]
+      }
+    }
+  ],
+  "analytics": {
+    "intents": [
+      {
+        "id": "base-quick-replies",
+        "description": "Quick Replies Messages",
+        "confidenceScore": 1
+      }
+    ]
+  }
+}
+```
+
+Figure 2.6 Showing an example of a Quick Replies response
+
+### Bot Actions
+
+{: .notice}
+Please note we only support **ONE ACTION** per Custom Endpoint service response
```

Comment:
Shouldn't we have this on every vendor? Or even better in the general section of the documentation

Reply:
I would still prefer it to have it on Actions to make it even more descriptive so that it's a kind of constant reminder to whoever is looking at a specific part of the document that he must keep it in mind.

Reply:
See my above comment about maintainability of the documentation.
If you stick with this please add it to all vendors

Reply:
will add new page